tests/freqtradebot/test_integration.py

```diff
@@ -685,3 +689,77 @@ def test_dca_exiting(default_conf_usdt, ticker_usdt, fee, mocker, caplog, levera
     else:
         # total won't change in futures mode, only free / used will.
         assert freqtrade.wallets.get_total("USDT") == starting_amount + trade.realized_profit
+
+
```

Comment:
In **test_dca_order_adjust_entry_replace_fails**  You used Trade.id as a filtering condition to get "trades". How do you know the supposed value of Trade.id? Is is always equal to the latest trade opened?
Note: I don't plan to modify this test, I'm just reading some test to have a better gaps of what is possible to do.
![image](https://github.com/user-attachments/assets/145e3ec5-f918-46b2-b553-d728997fca1f)

Reply:
`Trade.id` (if spelled like this, incl. capitalization) is not a value, but a sqlalchemy column used for the join.

If this is necessary here or not - i'd rather not comment (i think attached orders are loaded anyway so this probably could be simplified - but that's besides the point).

Reply:
Thank you for the clarification 👍🏽